src/editor/editor.cpp

```diff
@@ -669,6 +670,20 @@ Editor::event(const SDL_Event& ev)
 		redo();
 		}
 
+  if (ev.type == SDL_KEYDOWN &&
+    ev.key.keysym.mod & KMOD_SHIFT) {
+    m_scroll_speed = 96.0f;
+  }
+  else if (ev.type == SDL_KEYDOWN &&
```

Comment:
Using a nested if statement, you can avoid checking `ev.type == SDL_KEYDOWN` twice.

Reply:
While you're changing this part of the code  -  please format your `if` statemets as:
```cpp
if (condition)
{
  ...
}
else if (condition)
{
  ...
}
else
{
  ...
}
```

.. with all brackets of their own lines. Saves some code cleaning time :^)

Reply:
I didn't think of nested if statements, I'm working on it rn.
@Semphriss shouldn't I remove the brackets since the speed changes are one line only?

Reply:
Usually, brackets are omitted when the operation is trivial and the body of the `if` statement is practically guaranteed to never need additions (like `if (must_be_false) return;`).

With if-else statements, it's even more important since it's more likely to cause trouble with maintenance later (imagine someone adding a line when you didn't use brackets). For the moment, I'd advise keeping the brackets even though it's only one line, but I'm open to discussion.